server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -979,11 +980,14 @@ exports.updateClient = async function (body, user, xCorrelator, traceIndicator,
   );
   let logicalTerminationPointConfigurationStatus;
   if (httpClientUuid) {
-    logicalTerminationPointConfigurationStatus = await LogicalTerminationPointService.createOrUpdateApplicationLtpsAsync(
-      logicalTerminationPointConfigurationInput
+    logicalTerminationPointConfigurationStatus = await LogicalTerminationPointServiceMapping.findAndUpdateLogicalTerminationPointInstanceGroupExcludingOldReleaseAndNewReleaseAsync(
+      logicalTerminationPointConfigurationInput,httpClientUuid
     );
   }
-
+  if(currentApplicationName != futureApplicationName){
+    let applicationName = await httpClientInterface.setApplicationNameAsync(httpClientUuid, futureApplicationName)  
```

Comment:
error  'applicationName' is assigned a value but never used  no-unused-vars. please remove the "applicationName" reference.

Reply:
done